Vagrantfile

```diff
@@ -147,6 +159,14 @@ Vagrant.configure(2) do |config|
   end
 
   config.vm.provider "parallels" do |prl, override|
+    # Default box for parallels on arm64
```

Comment:
Same for this comment, it's just the "what" of this block, and it's simple enough already to not warrant it.